test_reporting/report_uploader.py

```diff
@@ -35,23 +35,29 @@ def _run_script():
     kusto_db = KustoConnector(args.db_name)
 
     if args.category == "test_result":
-        if args.json:
-            test_result_json = validate_junit_json_file(args.path_name)
-        else:
-            roots = validate_junit_xml_path(args.path_name)
-            test_result_json = parse_test_result(roots)
+        test_result_json = {}
+        for path_name in args.path_list:
+            if args.json:
+                test_result_json.update(validate_junit_json_file(path_name))
```

Comment:
I don't think this will work for multiple test reports b/c when we call update we will overwrite the metadata + summary each time.

Reply:
Good catch. I've changed it to upload one file/folder at a time, per our offline discussion.